.prow.sh

```diff
@@ -2,27 +2,59 @@
 
 . release-tools/prow.sh
 
-# External-provisioner is updated to use VolumeSnapshot v1 APIs. As a result,
-# tesing on Kubernetes version < 1.20 are no longer supported because they use
-# v1beta1 or earlier VolumeSnapshot APIs.
-# TODO: The check can be removed when all Prow jobs for Kubernetes < 1.20 are removed.
-if ! (version_gt "${CSI_PROW_KUBERNETES_VERSION}" "1.19.255" || [ "${CSI_PROW_KUBERNETES_VERSION}" == "latest" ]); then
-    filtered_tests=
-    skipped_tests=
-    for test in ${CSI_PROW_TESTS}; do
-	case "$test" in
-	    parallel | serial | parallel-alpha | serial-alpha)
-		skipped_tests="$skipped_tests $test"
-		;;
-	    *)
-		filtered_tests="$filtered_tests $test"
-		;;
-	esac
+volume_mode_conversion () {
+   [ "${VOLUME_MODE_CONVERSION_TESTS}" == "true" ]
+}
+
+if volume_mode_conversion; then
+  install_snapshot_controller() {
+      CONTROLLER_DIR="https://raw.githubusercontent.com/RaunakShah/external-snapshotter/${CSI_SNAPSHOTTER_VERSION}"
```

Comment:
CONTROLLER_DIR="https://raw.githubusercontent.com/RaunakShah/external-snapshotter/${CSI_SNAPSHOTTER_VERSION}"?

Can you fix this?  This is your private repo.

Reply:
Never mind. I see that this is fixed by the 2nd commit.